src/PlayerRespawn.java

```diff
@@ -10,17 +11,20 @@
 import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerRespawnEvent;
 import org.bukkit.scoreboard.Team;
+import org.bukkit.inventory.ItemStack;
 
 /** Monitors people respawning
  * If somebody participating in the event respawns, it teleports them home
  * and gives them back previous items and health, etc */
 public class PlayerRespawn implements Listener {
 
+	static public HashMap<Player , ItemStack[]> items = new HashMap<Player , ItemStack[]>();
```

Comment:
In general you shouldn't store Player objects, because that means you'll be holding on to the object even if the server isn't holding it anymore. Prefer to use UUIDs instead.